spring-integration-core/src/main/java/org/springframework/integration/config/EnableIntegrationManagement.java

```diff
@@ -62,4 +62,12 @@
 	 */
 	String defaultLoggingEnabled() default "true";
 
+	/**
+	 * Set simple pattern component names matching for observation registry injection.
+	 * @return simple pattern component names matching for observation registry injection.
+	 * @since 6.0
+	 * @see org.springframework.integration.support.utils.PatternMatchUtils#smartMatch(String, String...)
+	 */
+	String[] observationPatterns() default "*";
```

Comment:
I think I would make the default `""` - observation is quite expensive so I would suggest that users should opt-in.

Reply:
Well, you opted-in, when you have added this `@EnableIntegrationManagement` into your configuration.
If not -there is no any instrumentation at all.
So, in my feeling if you have added such a marker, then you probably would expect everything to be instrumented.
Otherwise it sounds double: add annotation and then add an `*` pattern to enable all 🤷

Reply:
I doubt whether anyone would ever want to instrument every component in an integration flow with observation. I would expect that, in most cases, they would only enable it on inbound and outbound endpoints.

Also, people might want simple timers (metrics) without the extra overhead of observation. Using observation for just timers is less performant.